subgraphs/starter-example/subgraph.template.yaml

```diff
@@ -0,0 +1,25 @@
+specVersion: 0.0.4
```

Comment:
Q: why 0.0.4? Just curious

Reply:
good question, I have been using that version, however just tested it out for latest `specVersion` = 0.0.5
and `apiVersion` = 0.0.7
And was sucessful, so better to use latest versions 👍 

✅